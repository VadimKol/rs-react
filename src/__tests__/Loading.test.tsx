import { render } from '@testing-library/react';

import Loading from '@/app/loading';

describe('Header Component', () => {
  it('renders correctly', () => {
    const { container } = render(<Loading />);

    expect(container).toMatchSnapshot();
  });
});
