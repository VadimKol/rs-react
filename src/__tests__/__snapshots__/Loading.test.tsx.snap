// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders correctly 1`] = `
<div>
  <main
    class="main"
  >
    <div
      class="loader"
    />
  </main>
</div>
`;
